meerk40t/gui/wxmribbon.py

```diff
@@ -403,8 +403,14 @@ def _button_click_id(self, evt_id, event=None):
             return
         button.toggle = not button.toggle
         if button.toggle:
+            if button.toggle_attr is not None:
+                setattr(button.object, button.toggle_attr, True)
+                self.context.signal(button.toggle_attr, True, button.object)
             self._restore_button_aspect(button, button.state_pressed)
         else:
+            if button.toggle_attr is not None:
+                setattr(button.object, button.toggle_attr, False)
+                self.context.signal(button.toggle_attr, False, button.object)
```

Comment:
We flip the states here. But then also send the button.object (which is needed for choice panels)